### ISSUE TITLE
Awe.js works fine on Chromebook and Win10 but not Android

### ISSUE THREAD

**Issue by theo-armour** [buildar/awe.js#12]
Hi Awe-some peeps

I have the awe.js marker demo working on a ChromeBook Pixel and a Win10 computer, but nothing shows up on my Google Nexus 6 or my Nvidia Shield Androd tablet.

What further information can I provide you to help come up with a solution?

Theo

**Comment by robman** [comment]
Hey Theo, it's probably being blocked by a current bug in the Chrome video/canvas handling. Here's the most likely culprit (but there are a few at the moment).
https://code.google.com/p/chromium/issues/detail?id=501208

There's also an issue with Firefox on Android where videos are rendered upside down - but that's fixed from version 40 onwards.

I'll leave this issue open as a notice for any other users and close it when that Chrome bug is fixed.

Thanks for raising the issue 8)

**Comment by theo-armour** [comment]
@robman Thanks for the speedy reply.

I have voted for the fix on code.google.com.

I confirm awe.js is working with FF on my Nvidia Shield tablet. Sadly, FF 40.0.3 is crashing when I load the demo marker script on my Google Nexus 6.

**Comment by robman** [comment]
Hi @theo-armour this should all be resolved now - please checkout our updated repos at https://github.com/awe-media/awe.js which includes new docs in the wiki.

If you find this doesn't work then please raise a new issue there.
